src/Digdir.Domain.Dialogporten.ChangeDataCapture/ChangeDataCapture/Checkpoints/CheckpointSyncronizer.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections.Concurrent;
+using Npgsql.Replication;
+
+namespace Digdir.Domain.Dialogporten.ChangeDataCapture.ChangeDataCapture.Checkpoints;
+
+internal sealed class CheckpointSyncronizer : BackgroundService, IAsyncDisposable
+{
+    private IReadOnlyCollection<Checkpoint> _syncedCheckpoints = new List<Checkpoint>().AsReadOnly();
+
+    private readonly PeriodicTimer _periodicTimer = new(TimeSpan.FromSeconds(10));
```

Comment:
Kanskje denne burde være konfigurerbar? 🤔

Reply:
Jeg stemmer for det, greit å kunne endre sånn i fart i tilfelle problemer